pymongo/encryption.py

```diff
@@ -260,15 +260,20 @@ def close(self):
 
 
 class RewrapManyDataKeyResult(object):
+    """Result object returned by a ``rewrap_many_data_key`` operation.
+
+    .. versionadded:: 4.2
+    """
+
     def __init__(self, bulk_write_result: Optional[BulkWriteResult] = None) -> None:
-        """Result object returned by a ``rewrap_many_data_key`` operation.
+        """Create a ``RewrapManyDataKeyResult`` for a ``rewrap_many_data_key`` operation.
```

Comment:
Following the pattern in the other result classes (eg UpdateResult) we can remove the `__init__` docstring. Users aren't expected to create this class anyway.

Reply:
Done